contracts/staking/contracts/src/stake/MixinStake.sol

```diff
@@ -188,7 +188,7 @@ contract MixinStake is
         _assertStakingPoolExists(poolId);
 
         // Cache amount delegated to pool by staker.
-        IStructs.StoredBalance memory initDelegatedStake =
+        IStructs.StoredBalance memory initDelegatedStakeToPoolByOwner =
```

Comment:
Nit: can we keep the `initDelegatedStake` name for this? `initDelegatedStakeToPoolByOwner` doesn't really describe this accurately (and also mirrors the name of the mapping, which is confusing imo).